src/components/Editor/Runners/HtmlRunner/HtmlRunner.js

```diff
@@ -20,19 +20,29 @@ function HtmlRunner() {
     // TODO: get html files and handle urls for non index pages
     var indexPage = projectCode[0].content;
 
-    var cssFiles = projectCode.filter(component => component.extension === 'css');
-    cssFiles.forEach(cssFile => {
-      var cssFileBlob = getBlobURL(cssFile.content, 'text/css');
-      indexPage = indexPage.replace(`href="${cssFile.name}.css"`, `href="${cssFileBlob}"`)
+    var cssFiles = projectCode.filter(
```

Comment:
how come we're using `var` and not `let` here?

Reply:
honestly no idea! This is another formatting change. I'd just spotted the styles were importing `css` over `scss` and wanted to correct it 😅

Reply:
haha, I don't know how you feel, but I don't think we should be mixing var with let and const, since it can lead to accidental side effects if you redeclare the variable? and none of the uses of var here need to be var, they could be refactored to let and const, and that would make the code a little more robust.

Reply:
probably something we can fix as we go along

Reply:
I can sort these out in this PR for sure, but definitely something to be mindful of. Really weird use case here 😅

Reply:
done 👍